query/src/main/java/org/infinispan/search/mapper/session/impl/InfinispanSearchSession.java

```diff
@@ -65,6 +65,11 @@ public String tenantIdentifier() {
       return null;
    }
 
+   @Override
+   public Object tenantIdentifierValue() {
+      return "";
```

Comment:
or should it be `null` ? ( I think in Search, we check for null tenant identifiers but not for the blank ones ...)